storageconnector/fixtures/fixtures.sql

```diff
@@ -7,4 +7,5 @@ INSERT INTO features VALUES (3, 'python3', 'python3', 'Olaf', '2016-11-25 11:41:
 INSERT INTO features VALUES (4, 'numpy', 'python2 package numpy', 'Olaf', '2016-11-25 11:43:50+01', 'python2', 'sudo apt install python-numpy');
 INSERT INTO features VALUES (5, 'numpy3', 'python3 package numpy', 'Olaf', '2016-11-25 11:44:40+01', 'python3', 'sudo apt install python3-numpy');
 INSERT INTO features VALUES (6, 'numpy-all', 'python2+python3 package numpy','Olaf', '2016-11-25 11:45:30+01    ', 'python2 python3', 'sudo apt install python-numpy && sudo apt install python3-numpy');
+INSERT INTO features VALUES (7, 'java-python2', 'python2 + java','Olaf', '2016-11-25 11:45:30+01    ', 'python2 java', '');
```

Comment:
btw, I've replaced the fixtures in #126 to make them compatible with what we have in Git